scripts/spectrum-tokens.js

```diff
@@ -119,7 +119,9 @@ const processPackages = async (srcPath, index) => {
         `\\s*--spectrum-alias-${packageName}[^;]*;`,
         'g'
     );
-    for (const varsPath of await fg([varsPaths])) {
+    for (const varsPath of await fg([varsPaths], {
+        ignore: ['**/tokens/**/*.css'],
+    })) {
```

Comment:
Consider pulling this out?

```suggestion
    const varsWithoutTokens = await fg([varsPaths], {
        ignore: ['**/tokens/**/*.css'],
    });
    for (const varsPath of varsWithoutTokens) {
```